src/db/sysdb_ops.c

```diff
@@ -2023,12 +2018,19 @@ int sysdb_add_user(struct sss_domain_info *domain,
 /* =Add-Basic-Group-NO-CHECKS============================================= */
 
 int sysdb_add_basic_group(struct sss_domain_info *domain,
-                          const char *name, gid_t gid)
+                          const char *name,
+                          bool is_posix,
+                          gid_t gid)
 {
     struct ldb_message *msg;
     int ret;
     TALLOC_CTX *tmp_ctx;
 
+    if (is_posix && gid == 0) {
+        DEBUG(SSSDBG_MINOR_FAILURE, "POSIX groups with gid==0 are not supported.\n");
```

Comment:
Can this really happen? For example, if gid=0 is set in LDAP server?

If so then I'd log group name and maybe would use `SSSDBG_IMPORTANT_INFO`

Or does it indicate a bug in SSSD?

Reply:
Then I would use `SSSDBG_OP_FAILURE` at least to get a backtrace by default.